lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -0,0 +1,174 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone',
+        'js/student_account/models/user_account_model',
+        'js/student_account/models/user_preferences_model',
+        'js/student_account/views/account_settings_fields',
+        'js/student_account/views/account_settings_view',
+    ], function (gettext, $, _, Backbone, UserAccountModel, UserPreferencesModel, FieldViews, AccountSettingsView) {
+
+        return function (fieldsData, userAccountsApiUrl, userPreferencesApiUrl) {
+
+            var accountSettingsElement = $('.wrapper-account-settings');
+
+            var userAccountModel = new UserAccountModel();
+            userAccountModel.url = userAccountsApiUrl;
+
+            var userPreferencesModel = new UserPreferencesModel();
+            userPreferencesModel.url = userPreferencesApiUrl;
+
+            var sectionsData = [
+                 {
+                    title: gettext('Basic Account Information'),
+                    fields: [
+                        {
+                            view: new FieldViews.ReadonlyFieldView({
+                                model: userAccountModel,
+                                title: gettext('Username'),
+                                valueAttribute: 'username',
+                                helpMessage: ''
+                            })
+                        },
+                        {
+                            view: new FieldViews.TextFieldView({
+                                model: userAccountModel,
+                                title: gettext('Full Name'),
+                                valueAttribute: 'name',
+                                helpMessage: gettext('This is the name used on your edX certificates. Changes to this field are reviewed.')
```

Comment:
@symbolist @explorerleslie I think the full name is only used for certificates - correct? If so, let's go with this:

**The name that appears on your edX certificates.**